src/main/res/values/strings.xml

```diff
@@ -45,23 +45,23 @@
   <string name="downloading_format">Downloading %s</string>
   <string name="edit_repository">Edit repository</string>
   <string name="empty_search_summary">No matching applications found</string>
-  <string name="error_downloading_format">Error downloading %s</string>
-  <string name="error_syncing_format">Error syncing %s</string>
-  <string name="error_validating_format">Error validating %s</string>
+  <string name="error_downloading_format">Could not download %s</string>
+  <string name="error_syncing_format">Could not sync %s</string>
+  <string name="error_validating_format">Could not validate %s</string>
   <string name="fingerprint">Fingerprint</string>
-  <string name="finished_downloading_format">Finished downloading %s</string>
+  <string name="finished_downloading_format">Downloaded %s</string>
   <string name="http_error_description">Invalid server response.</string>
   <string name="http_proxy">HTTP proxy</string>
   <string name="ignore_all_updates">Ignore all updates</string>
   <string name="ignore_this_update">Ignore this update</string>
   <string name="incompatible_features_description">Missing features:</string>
-  <string name="incompatible_older_description">This version is older than installed on your device.
-    You\'ll have to uninstall the application to proceed.</string>
+  <string name="incompatible_older_description">This version is older than the one installed on your device.
+    Uninstall that first.</string>
```

Comment:
There is another string "This version is signed with a different certificate. You'll have to uninstall the application to proceed." which ends with "You'll have to uninstall the application to proceed.". I think either of them (or both of them) should be changed so they ended similarly.

Reply:
I made it work, sort of. Might be that it is possible to boil it into one sentence, but I want to reconsider more closely before suggesting anything again :) You have good patience, and I should have done better.